### PR TITLE
Add getLodgePrices endpoint to fetch lodge prices

### PR DESCRIPTION
**Pull request by jeffplays2005**

### CODE REVIEW

**Comment by jeffplays2005** [comment]
Can't perform testing as don't have access to Stripe key

**Comment by jeffplays2005** [comment]
Conducted some manual testing 
image

File: server/src/service-layer/controllers/PaymentController.ts [hunk 1]
```diff
@@ -107,6 +109,58 @@ export class PaymentController extends Controller {
...
+        switch (lodgeType) {
+          case LodgePricingTypeValues.SingleFridayOrSaturday: {
+            name = LodgePricingTypeValues.SingleFridayOrSaturday
+            break
+          }
+          case LodgePricingTypeValues.Normal: {
+            name = LodgePricingTypeValues.Normal
+            break
+          }
+        }
```
Comment (bcho892) [inline server/src/service-layer/controllers/PaymentController.ts:140]: nit: i know most logic here is copy pasted but this can be inlined

**Review by bcho892 — APPROVED** [review]

File: server/src/service-layer/response-models/PaymentResponse.ts [hunk 2]
```diff
@@ -31,6 +34,18 @@ export interface MembershipStripeProductResponse extends CommonResponse {
...
+// Make a data shape matching to the expected response from Stripe API
+export interface LodgeStripeProductResponse extends CommonResponse {
+  data?: {
+    productId: string
+    name: LodgePricingTypeValues
+    description?: string
+    discount: boolean
+    displayPrice: string
+    originalPrice?: string
+  }[]
```
Comment (bcho892) [inline server/src/service-layer/response-models/PaymentResponse.ts:46]: nit: documentation of each field

File: server/src/service-layer/controllers/PaymentController.ts (hunk 1 above)
Comment (bcho892) [reply server/src/service-layer/controllers/PaymentController.ts:140]: what i mean is just set the `name: lodgeType`

**Pull request merged**